### PR TITLE
Awesome books- Collaboration

### PR DESCRIPTION
**Pull request by Patfarmurs**
Created the awesome book page with : 
JavaScript
HTML
CSS

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -38,19 +38,15 @@ ManageBooks.prototype.delete = function (e) {
   this.display();
 };
```
Comment (GzimAsani) [inline index.js:39]: - [ ] I am getting an error when trying to add a book
image
_It is comming from index.js on line 22_

- [ ] After fixing this make sure you have met other requirements as well getting the data from local storage

File: README.md [hunk 2]
```diff
@@ -249,4 +249,5 @@ This project is [MIT](./LICENSE) licensed.
 
 _NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
\ No newline at end of file
+
```
Comment (GzimAsani) [inline README.md]: - [ ] This pic is broken, kindly update it 
image

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello  @Patfarmurs  and @Ramaunspoken                        :wave:   
 Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job using the Microverse readme template :+1: 

 ## Required Changes 
- [ ] Check the comments below :arrow_double_down: 



 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js (hunk 1 above)
Comment (Ramaunspoken) [reply index.js:39]: @GzimAsani
The Local storage is working just fine.

**Review by mckent05 — APPROVED** [review]
Hi @Patfarmurs,

# STATUS: APPROVED ✔️ ✔️  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations and well done! 🎉 
approved


## Highlights
- All linter checks are passing ✔️ 
- Your project works as expected 👏 
- Your PR is professional 👍 👍 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**